packages/discovery-styles/scss/components/search-results/_search-results.scss

```diff
@@ -175,6 +175,7 @@ $result-table-fade-offset: calc(#{$result-element-max-height} - #{$result-table-
 }
 
 .#{$prefix}--search-result__footer__title {
+  max-width: 75%;
```

Comment:
Why is this necessary?

Reply:
If the max width of the footer title is not constrained in any way, it will actually occupy almost all of the footer space leaving no room for the collection name on the right side.

Reply:
Changed to `flex: 1` and added extra margin on right of footer title.